scipy/stats/_sobol.pyi

```diff
@@ -0,0 +1,58 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import numpy.typing as npt
```

Comment:
```suggestion
import numpy.typing as npt
```
FYI, `if TYPE_CHECKING:` checks are redundant within .pyi stub files, 
as the expression _always_ evaluates to `True` in such cases.

Reply:
oh thanks, we should then remove this from the other `pyi` files. This is something you could do here @akpandeya (optional of course).

Reply:
> Side note: sorry we did not have time to go through your point! I added it to the next topic list.

Yeah, it was a _big_ list though, so not a huge suprise.

Reply:
ok.  Thanks for the comment. Will improve the import.